stream_alert/classifier/payload/payload_base.py

```diff
@@ -62,14 +62,24 @@ def __len__(self):
         )
 
     def __repr__(self):
+        record_data = self._record_data
+        invalid_records = self.invalid_records
+
         try:
-            record_data = json.dumps(self._record_data)
-            invalid_records = json.dumps(self.invalid_records)
+            if isinstance(record_data, dict):
+                record_data = json.dumps(record_data)
         except (TypeError, ValueError):
-            record_data = self._record_data
-            invalid_records = self.invalid_records
             LOGGER.debug('A PayloadRecord has data that is not serializable as JSON')
 
+        try:
+            #Build a JSON string if data is a dict or non-empty list of dicts
```

Comment:
1 space after `#` in comments

Reply:
@Ryxias Do you know if you can setup pylint or IDE to detect or automatically add the space?